aten/src/ATen/cuda/CUDAGraph.h

```diff
@@ -1,19 +1,27 @@
+#pragma once
+
 #include <ATen/Tensor.h>
 #include <c10/core/Device.h>
 #include <c10/cuda/CUDAStream.h>
-#include <ATen/CUDAGeneratorImpl.h>
+
+// global typedef but i don't see any cops
+typedef unsigned long long CUDACaptureid_t;
```

Comment:
🚨🚨 why not use a namespaced `using` declaration? 🚨🚨

Reply:
(Answer is probably something like, you could get this from CUDA headers, but mumble mumble portability? Say that.)

Reply:
lol

Reply:
I'll use a namespaced using declaration, there's no reason not to. The cuda graph API explicitly wants an `unsigned long long`  for the graph id.

Reply:
done